node-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,165 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'node',
+        rules: [
+          {
+            alert: 'NodeFilesystemSpaceFillingUp',
+            expr: |||
+              predict_linear(node_filesystem_avail{%(nodeExporterSelector)s,%(fsSelectors)s}[6h], 24*60*60) < 0
+                AND
+              node_filesystem_avail{%(nodeExporterSelector)s,%(fsSelectors)s} / node_filesystem_size{%(nodeExporterSelector)s,%(fsSelectors)s} < 0.4
+                AND
+              node_filesystem_readonly{%(nodeExporterSelector)s,%(fsSelectors)s} == 0
+            ||| % $._config,
+            'for': '1h',
+            labels: {
+              severity: 'warning',
```

Comment:
Our convention is page and ticket for severity

Reply:
There was a long debate at some point as to what to use in our examples, which I can't find now. Page and ticket won out in the end.

Reply:
They didn't won, there was no agreement on such convention and I closed the ticket as it wasn't that important to me to change the one example where you used it.

Reply:
There was some discussion at the DevSummit in Munich. I read through the doc and couldn't find anything more specific other than: we want a default. Are critical and warning our defaults now?

Reply:
Yes

Reply:
FTR: Personally, I like "page" and "ticket" most, but I have seen "critical" and "warning" a lot, and we have also used it in the mixins in prometheus/prometheus. Let's stick with it for now. The beauty of jsonnet is that everybody can change those very easily in one line of code.